Bash-Email script

manb-msft opened the pull request:
This is a simple post-build script for a bash email, along with a readme.md

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Microsoft/appcenter-build-scripts-examples?pullRequest=4) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Microsoft/appcenter-build-scripts-examples?pullRequest=4) before we can accept your contribution.<br/><br/>:x: manb-msft [sign now](https://cla.opensource.microsoft.com/Microsoft/appcenter-build-scripts-examples?pullRequest=4)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Microsoft/appcenter-build-scripts-examples?pullRequest=4) it.</sub>

derpixeldan commented:
This is great, can you please incorporate the README in the script itself as comments, it's more on point in case people miss it.

derpixeldan commented:
And add an entry to the table of content in the root README.

manb-msft commented:
@derpixeldan Thanks! Makes sense. Modified the file and removed the readme. Also moved it to the general folder. 
Sorry about too many smaller commits (Rookie mistake)
